plan/validator.go

```diff
@@ -302,3 +307,49 @@ func checkDuplicateColumnName(indexColNames []*ast.IndexColName) error {
 	}
 	return nil
 }
+
+// checkFieldLengthLimitation check the maximum length of the column
```

Comment:
s/check/checks
Add `.` at the end of a line. And please update the similar comments.